docs/io/fits/usage/unfamiliar.rst

```diff
@@ -26,6 +26,8 @@ table, Astropy will automatically detect what kind of table it is.
     >>> from astropy.io import fits
     >>> filename = fits.util.get_testdata_filepath('ascii.fits')
     >>> hdul = fits.open(filename)
+
+.. doctest-requires:: numpy<=1.14.1
```

Comment:
Apparently this doesn't actually work, because I got the condition wrong but the test is skipped anyway...

Reply:
I should probably just change it to `.. doctest-skip` for now. It's looking for a module called `numpy<=1.14.1` which of course is going to cause it to be skipped.

Reply:
So is that a bug or feature? If a bug, let's open an issue, as that is a battle for another day...

Reply:
It's not a bug, it's just a missing feature: https://github.com/astropy/pytest-doctestplus/issues/18